dynawo/sources/Models/Modelica/Dynawo/Examples/Wind/IEC/Neplan/WPP4ACurrentSource.mo

```diff
@@ -18,7 +18,7 @@ model WPP4ACurrentSource "Wind Power Plant Type 4A model from IEC 61400-27-1 sta
 
   extends Icons.Example;
 
-  Dynawo.Electrical.Events.NodeFault nodeFault1(RPu = 0, XPu = 0.379, tBegin = 12, tEnd = 12.15) annotation(
+  Dynawo.Electrical.Events.NodeFault nodeFault1(RPu = 0, XPu = 0.4, tBegin = 12, tEnd = 12.15) annotation(
```

Comment:
It starts to get a bit high there, what doesn't work with a frank fault ?

Reply:
At the end of the fault, argument of sqrt is < 0

Reply:
Ok but does the model itself prevent IMax² - ipcmd² < 0 ? Is it just a solver issue or a model issue ?

Reply:
Yes, with max(0, IMax² - ipcmd²), so solver issue